### DISCUSSION TITLE
VDES in the Netherlands, Belgium, Germany

### DISCUSSION THREAD

**Discussion by muebau** [peterantypas/maiana#141] [General]
Hello,
I just stumbled about the new requirements coming with VDES. There is a real chance that AIS would expanded to more channels in 2028. I talked today with some regulatory entity and the plans are very vague. Truth is that starting from 01.01.2024 channels 24, 25, 26, 27, 28, 2027, 2028, 84, 85 and 86 are blocked for VDES in the Netherlands and Belgium (Germany stepped back in the last moment). It is not allowed to use these channels anymore and fines might apply.

Does somebody have more information than me?
Does this affect MAIANA?

**Comment by matthew-quirke** [comment]
**Hi Muebau,**  very interesting info!
From what I’ve read in the new VDES standard, it looks like it’s designed to be backward compatible and support coexistence with legacy AIS systems — including Class A, Class B, and Class B+. Is that how you understand it as well? So no doesn't' affect miaana -

**Comment by muebau** [comment]
Hello Matthew,
Yes it seems very sensible to me that it is no braking change. It would render all other AIS systems useless too. We need to keep an eye on the fact that possible changes of the AIS channel can be announced via DSC according to the standard. MAIANA would be unaware of this as it does not process DSC messages.

It could be very interesting to see if we could extend the MAIANA firmware to cover (possible) new features coming with the change.

**Reply by marcelrv** [reply]
Interesting indeed... As the new VDES functionality appears to use same frequencies (or at least same attenna, hence prob freq) and MAIANA is ultimately a 2 way software driven radio, I can imaging with the right additions to the firmware it technically could support it.
